src/media/js/reviews.js

```diff
@@ -213,50 +213,55 @@ define('reviews',
         var $this = $(this);
         var data = utils.getVars($this.serialize());
         var slug = data.app;
-        forms.toggleSubmitFormState($this);
-
-        requests.post(urls.api.url('reviews'), data).done(function(new_review) {
-            // Update the ratings listing for the app.
-            rewriter(slug, function(data) {
-                data.objects.unshift(new_review);
-                data.meta.total_count += 1;
-                if (!data.user) {
-                    data.user = {};
-                }
-                data.user.has_rated = true;
-                return data;
-            });
-
-            var new_rating = parseInt(data.rating, 10);
-
-            // Update the app model.
-            var app_model = models('app').lookup(slug);
-            if (app_model) {
-                var num_ratings = app_model.ratings.count;
-                if (!num_ratings) {
-                    app_model.ratings.average = new_rating;
-                } else {
-                    // Update the app's rating to reflect the new average.
-                    app_model.ratings.average =
-                        (app_model.ratings.average * num_ratings + new_rating) /
-                        (num_ratings + 1);
+        if(data.text.trim().length > 0)
```

Comment:
we don't have to wrap *everything* in the if statement. it's sufficient to just ```return```. also you can view mozilla's webdev style guide: https://mozweb.readthedocs.org/en/latest/reference/js-style.html